### PR TITLE
Add seeds data

### LINKED ISSUE #43: Add test data to seeds.rb
So that we can prepare dummy dictionary entry quickly on local

### PR DESCRIPTION
**Pull request by Ko001**
Fixes #43

### CODE REVIEW

**Review by ryonakano — CHANGES_REQUESTED** [review]
I guess you shouldn't remove these files:

* app/assets/stylesheets/search.scss
* app/javascript/packs/application.js
* app/views/search/show.html.erb

**Comment by Ko001** [comment]
Thanks for your PR.
It was totally an accident.  I didn't mean to do this, my apologies.
I restored those files.

**Review by ryonakano — COMMENTED** [review]
I don't think adding `db/migrate/20201022035429_add_alt_answer_to_quiz_table.rb` is not related to this PR. I guess you'd like to add columns for alternative answers but it would be better to create another PR for this.

Keeping unrelated things out makes PRs smaller, cleaner, and easier to revert in the future if needed.

**Comment by Ko001** [comment]
Well, you are absolutely correct.  I somehow messed up this branch and add-js-for-quizzes.  The file is for answer check function.

**Comment by Ko001** [comment]
Please forget about my deleted comment.
I understand your point.  However, the file seems necessary to me, though I'm not sure if we have to add it to this PR.

Currently, the answer for the last post quiz is like this below.
`Quiz.create(original_noun: 'sale', sentence_jp: 'シャロンはセールで安く選んだとてもすてきな家具をもっている。', sentence_en: 'Sharon has some really nice furniture she picked up cheap in ( sale ).', placeholder: 'sale', answer: 'a [the] sale', kind: 'post')`

It has two answers but only one answer column on the table.  It has made me think of adding alternative answers column on this branch and changing answer check function in accordance with it.
I know I'm sort of messing up this branch and add-js-for-quizzes branch, so I wanna have your opinion, again.

**Comment by ryonakano** [comment]
Ah now I see the problem and the reason why you added it… but it would be another issue and better to fix in another PR. I opened an issue for that (#45) so we may want to revert adding this file

**Comment by Ko001** [comment]
100% agree.  And sorry again, I made a mess.  I just learned how to revert...

**Comment by ryonakano** [comment]
Two concerns I have:

- is it OK to quote example sentences in some dictionaries without credit?—maybe not.
- also, suppose we credit, is it OK to use these dictionaries like this? I mean, some dictionaries may only allow for personal use like just checking words meanings or quoting the contents in our reports and forbid publicizing quoted contents.

We may want to ask Hikky about these thing.

**Comment by Ko001** [comment]
100% agree, we need to ask him.

It might be possible to claim this dictionary is a part of his research and that the best way to show word categorization, though we may have to delete quizzes.
It's just a ramdom thougt.

File: app/models/quiz.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class Quiz < ApplicationRecord
+  belongs_to :nouns
```
Comment (ryonakano) [inline app/models/quiz.rb:2]:
```suggestion
```

File: app/models/noun.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 class Noun < ApplicationRecord
   has_many :noun_rows, dependent: :destroy
+  has_many :quizzes, dependent: :destroy
```
Comment (ryonakano) [inline app/models/noun.rb:3]:
```suggestion
```

**Review by ryonakano — CHANGES_REQUESTED** [review]
Sorry for the late review! (FYI I modified your PR to unify the use of `"` and `'` :wink:)

After some testing with this branch, I suppose we don't need the following changes to the models. I got the following errors when I try to create a new Quiz instance:

```ruby
@messages={:nouns=>["must exist"]}, @details={:nouns=>[{:error=>:blank}]}>
```

Removing these two lines creates a new pre/post quiz successfully.

**Review by ryonakano — APPROVED** [review]
Works fine.

**Pull request merged**